src/Network/DogStatsd.hs

```diff
@@ -0,0 +1,53 @@
+module Network.DogStatsd (
+ dogStatsdClient
+, increment
+, decrement
+, count
+, gauge
+, timing
+, histogram
+) where
+
+import Network.Statsd (Stat, Type(..), fmtDatagram)
+import Network.Statsd.UdpClient(UdpClient(..), fromURI, send)
```

Comment:
I think we can remove the field import (..) from UdpClient as it doesn’t export anything beyond the type identifier.